korean/restroom/restroom.md

```diff
@@ -0,0 +1,11 @@
+화장실에서 할 일을 다 마쳤어요. 가볍고 시원한 기분으로 무엇을 할까요?
+
+[국회로 간다](filibuster/filibuster.md)?
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.